2025-03-17 12:23:08,073 - root - ERROR - 捕获到异常: division by zero
Traceback (most recent call last):
  File "/home/wfml/4ad/train.py", line 10, in <module>
    1 / 0  # 故意抛出异常（除以零）
    ~~^~~
ZeroDivisionError: division by zero
2025-03-17 12:23:14,598 - root - ERROR - 捕获到异常: division by zero
Traceback (most recent call last):
  File "/home/wfml/4ad/train.py", line 10, in <module>
    1 / 0  # 故意抛出异常（除以零）
    ~~^~~
ZeroDivisionError: division by zero
2025-03-17 12:23:22,274 - root - ERROR - 捕获到异常: division by zero
Traceback (most recent call last):
  File "/home/wfml/4ad/train.py", line 10, in <module>
    1 / 0  # 故意抛出异常（除以零）
    ~~^~~
ZeroDivisionError: division by zero
